nodejs/repositories.bzl

```diff
@@ -166,6 +167,9 @@ def _download_node(repository_ctx):
         return
 
     node_urls = repository_ctx.attr.node_urls
+    if not node_urls:
```

Comment:
don't you also want to check `len(node_urls)` per the commit message?

Reply:
`if not []:` is falsy